_people/1550_jenn-noinaj.md

```diff
@@ -0,0 +1,45 @@
+---
+# Page template info (DO NOT EDIT)
+layout: default
+banner_file: banner--people-projects-lg.svg
+banner_file_mobile: banner--people-projects-sm.svg
+people_page: true
+
+# Page link (Edit this)
+permalink: people/jenn-noinaj
+
+# People Carousel and Page (Edit this)
+name: Jenn Noinaj
+pronouns: She/Her
+alumni: false
+cop: Design
+city_of_origin: Chicago, IL
+previously_at: National Instruments
+image_name: jenn-noinaj.jpg
+quote: "USDS’ value is in delivery and building capacity. We’re able to work on projects to take an idea to implementation, and guide teams on best practices for how to continue this momentum even after we’re gone."
+
+---
+
+**What’s your background?**
+
+I’ve been in tech for the past ten years, and have really focused on working in industries and with organizations that aren’t familiar with user experience and product design. Most of my roles have been in standing up design best practices and processes while also delivering products and services that meet users’ needs.
+
+**What inspired you to join USDS?**
+
+When I was living in Austin, I was volunteering for the local Code for America brigade and doing a lot of community organizing for my neighborhood on local issues. I believe in government services that are people-centered, are inclusive, and that work for the people who need to use them. When I heard about USDS, specifically about healthcare.gov, the efforts being made to digitize the immigration process, and the work for veterans, I jumped at the chance to join and make a difference at the federal level.
+
+**What has been your biggest challenge?**
+
+My biggest challenge has been balancing the solutions that we propose with the organization’s appetite and willingness to embrace change. Even though I’m eager to roll out new initiatives to solve for all the things, I’ve learned to take a step back and determine what solutions make sense for the existing culture. Sometimes, that means that we have to do things incrementally. It’s more important that our work is sustainable, so I work with our government partners to understand where they’re coming from and solution together to find the right path forward.
+
+**How does your work make an impact?**
+
+I’ve been fortunate to work on a lot of different projects at USDS, and I think USDS’ value is in delivery and building capacity. We’re able to work on projects to take an idea to implementation, and guide teams on best practices for how to continue this momentum even after we’re gone. In terms of my work, this can mean firefighters get funding for equipment faster or helping federal agencies find and hire qualified talent.
```

Comment:
```suggestion
I’ve been fortunate to work on a lot of different projects at USDS, and I think USDS’ value is in delivery and building capacity. We’re able to work on projects to take an idea to implementation, and guide teams on best practices for how to continue this momentum even after we’re gone. In terms of my work, this can mean firefighters get funding for equipment faster, or federal agencies can find and hire qualified talent.
```